package.json

```diff
@@ -27,8 +27,7 @@
     "precommit": "lint-staged",
     "storybook": "start-storybook --port 6006",
     "storybook:build": "build-storybook",
-    "storybook:serve": "serve storybook-static",
-    "build-storybook": "yarn storybook:build"
```

Comment:
why did we have that duplication here 😅

Reply:
old open pull requests that were relying on it